README.md

```diff
@@ -97,9 +98,54 @@ See [here](https://openvinotoolkit.github.io/openvino_notebooks/?search=Create+a
 
 </details>
 
+## Performing visual language text generation
+<details>
+
+For more examples check out our [LLM Inference Guide](https://docs.openvino.ai/2024/learn-openvino/llm_inference_guide.html)
+
+### Converting and compressing the model from Hugging Face library
+
+```sh
+optimum-cli export openvino --model openbmb/MiniCPM-V-2_6 --trust-remote-code MiniCPM-V-2_6
+```
+
+### Run generation using VLMPipeline API in Python
+
+```python
+import openvino_genai as ov_genai
+#Will run model on CPU, GPU
+pipe = ov_genai.VLMPipeline("./MiniCPM-V-2_6/", "CPU")
```

Comment:
the comment looks confusing. You create pipeline on CPU, while comments states it will run on CPU / GPU

Reply:
maybe better to add device variable before pipeline and pass it instead of string?

Reply:
Updated